authorization/src/lib.rs

```diff
@@ -5,39 +5,37 @@ extern crate std;
 
 mod test;
 
-use soroban_sdk::{contractimpl, contracttype, BigInt, Env, IntoVal, Symbol};
+use soroban_sdk::{contractimpl, contracttype, symbol, BigInt, Env, IntoVal};
 use soroban_sdk_auth::{
     check_auth, NonceAuth, {Identifier, Signature},
 };
 
-#[derive(Clone)]
 #[contracttype]
 pub enum DataKey {
-    Acc(Identifier),
+    SavedNum(Identifier),
```

Comment:
It's unclear what `Acc` means to me, but it looked to me like that's the data key that the contract saves the number into, so I suggest we name the data key to indicate the intent of what data is being stored in it.